chain/cosmos/chain_node.go

```diff
@@ -653,9 +653,8 @@ func (tn *ChainNode) CreateNodeContainer() error {
 	chainCfg := tn.Chain.Config()
 	cmd := []string{chainCfg.Bin, "start", "--home", tn.NodeHome(), "--x-crisis-skip-assert-invariants"}
 	tn.logger().
-		WithField("container", tn.Name()).
-		WithField("command", strings.Join(cmd, " ")).
-		Info()
+		With("container", tn.Name()).
+		Info(strings.Join(cmd, " "))
```

Comment:
The previous version was more appropriate, maybe with a fixed message of "Running command".

Reply:
The way zap prints console output, I felt it was a better user experience seeing the command in the mssage vs. trying to find it in a field. But that is just my preference. For filtering/searching, it's probably better in a field.